frontend/src/components/generic/LockIcon.vue

```diff
@@ -1,8 +1,12 @@
 <template>
   <v-tooltip :disabled="hideTooltip" bottom>
     <template #activator="{ on }">
-      <v-icon v-if="value" small v-on="on"> mdi-lock-open-variant </v-icon>
-      <v-icon v-else small color="grey" v-on="on"> mdi-lock </v-icon>
+      <v-icon v-if="value" small v-on="{ click: iconClick, ...on }">
```

Comment:
using the native `dblclick` event didn't work?

Reply:
thx for the input